dpp/src/errors/codes.rs

```diff
@@ -92,6 +93,7 @@ impl ErrorWithCode for BasicError {
             Self::InvalidJsonSchemaRefError { .. } => 1014,
             Self::InconsistentCompoundIndexDataError { .. } => 1021,
             Self::IndexError(ref e) => e.get_code(),
+            Self::IdentityNotFoundError { .. } => 2000,
```

Comment:
Don't we have already something like that? For top up for example?

Reply:
This is the error from original `dpp` don't understand what you mean.